client/my-sites/site-settings/form-general.jsx

```diff
@@ -697,20 +690,20 @@ const connectComponent = connect( ( state ) => {
 	const siteId = getSelectedSiteId( state );
 
 	return {
-		isUnlaunchedSite: isUnlaunchedSite( state, siteId ),
-		isComingSoon: isSiteComingSoon( state, siteId ),
-		siteIsJetpack: isJetpackSite( state, siteId ),
-		siteIsVip: isVipSite( state, siteId ),
-		siteSlug: getSelectedSiteSlug( state ),
-		selectedSite: getSelectedSite( state ),
-		isPaidPlan: isCurrentPlanPaid( state, siteId ),
-		siteDomains: getDomainsBySiteId( state, siteId ),
-		isWPForTeamsSite: isSiteWPForTeams( state, siteId ),
-		isP2HubSite: isSiteP2Hub( state, siteId ),
 		customizerUrl: getCustomizerUrl( state, siteId, 'identity' ),
+		hasNoWpcomBranding: siteHasFeature( state, siteId, WPCOM_FEATURES_NO_WPCOM_BRANDING ),
 		isAtomicAndEditingToolkitDeactivated:
 			isAtomicSite( state, siteId ) &&
 			getSiteOption( state, siteId, 'editing_toolkit_is_active' ) === false,
+		isComingSoon: isSiteComingSoon( state, siteId ),
+		isP2HubSite: isSiteP2Hub( state, siteId ),
+		isPaidPlan: isCurrentPlanPaid( state, siteId ),
+		isUnlaunchedSite: isUnlaunchedSite( state, siteId ),
+		isWPForTeamsSite: isSiteWPForTeams( state, siteId ),
+		selectedSite: getSelectedSite( state ),
+		siteDomains: getDomainsBySiteId( state, siteId ),
+		siteIsJetpack: isJetpackSite( state, siteId ),
+		siteSlug: getSelectedSiteSlug( state ),
```

Comment:
Added: `hasNoWpcomBranding: siteHasFeature( state, siteId, WPCOM_FEATURES_NO_WPCOM_BRANDING ),`
Removed: `siteIsVip: isVipSite( state, siteId )`
All others: Sorted in alphabetical order